app/src/main/java/org/simple/clinic/bp/BloodPressureMeasurement.kt

```diff
@@ -2,32 +2,18 @@ package org.simple.clinic.bp
 
 import android.arch.persistence.room.Dao
 import android.arch.persistence.room.Entity
-import android.arch.persistence.room.ForeignKey
 import android.arch.persistence.room.Index
 import android.arch.persistence.room.Insert
 import android.arch.persistence.room.OnConflictStrategy
 import android.arch.persistence.room.PrimaryKey
 import android.arch.persistence.room.Query
 import io.reactivex.Flowable
 import org.simple.clinic.bp.sync.BloodPressureMeasurementPayload
-import org.simple.clinic.facility.Facility
 import org.simple.clinic.patient.SyncStatus
 import org.threeten.bp.Instant
 import java.util.UUID
 
-@Entity(
-    foreignKeys = [
-      ForeignKey(
-          entity = Facility::class,
-          parentColumns = ["uuid"],
-          childColumns = ["facilityUuid"],
-          onDelete = ForeignKey.CASCADE,
-          onUpdate = ForeignKey.CASCADE)
-    ],
-    indices = [
-      Index("patientUuid", unique = false),
-      Index("facilityUuid", unique = false)
-    ])
+@Entity(indices = [Index("patientUuid", unique = false)])
```

Comment:
We might want to consider keeping the index on `facilityUuid` while dropping the foreign key, depending on how frequently we query the table based on this field.

Reply:
We don't query BPs based on facilities right now, so should be fine.